src/tools/annotation/FreehandMouseTool.js

```diff
@@ -767,10 +790,10 @@ export default class FreehandMouseTool extends BaseAnnotationTool {
     const config = this.configuration;
     const currentTool = config.currentTool;
 
-    const dataHandles = toolState.data[currentTool].handles;
+    const dataHandles = toolState.data[currentTool].handles.dataHandles;
 
     // Don't allow the line being modified to intersect other lines
-    if (dataHandles.invalidHandlePlacement) {
+    if (toolState.data[currentTool].handles.invalidHandlePlacement) {
```

Comment:
We should use the cached value from line 793 here.

Reply:
the value in 793 is for the "dataHandles/points" array. This is accessing the handles object. we can still cache it though and use it for line 814

Reply:
Aha, sorry I missed that, I guess this highlights the `handles.dataHandles` confusion haha. But yeah, caching it for line 814 is a good shout 👍 .

Reply:
done